CONTRIBUTING.md

```diff
@@ -17,4 +17,9 @@ To start hacking, do the following:
 
 Now you're ready to start local development!
 
+### Caution
```

Comment:
This section should be moved to the bottom so it won't look like `export OG_HTML_DEBUG=1` is part of the Caution section.

Also you can link to this PR: https://github.com/zeit/og-image/pull/80